Automattic/fresh-data
# Add dev info summary data accessible from console

This adds developer info which is available at the `freshData` global,
which gives a single string summary, a list of resources with summaries
and detailed information, and a list of components that are currently
requiring resources.

To Test:

1. `npm install`, `npm run bootstrap`, `npm run build`
2. `cd packages/framework`, `npm link`, `npm run build`
3. From the `woocommerce-admin` repo, check out the `add/wc-api-dev-info` branch.
3. From `woocommerce-admin` repo, `npm install`, `npm link @fresh-data/framework`, `npm start`
4. Open an analytics page in wp-admin.
5. Open the JS dev console in your browser dev tools, type in `freshData.wcApi` and view the dev info data there. Ensure that the summary, resources, and components look correct.

Pro-tip: In Chrome dev tools, click the 
![DevTools_-_two_wordpress_test_wp-admin_admin_php_page_wc-admin](https://user-images.githubusercontent.com/945228/55422436-3f9d0100-5541-11e9-8365-d9d4d021be7d.png) to add `freshData.wcApi.summary` as a "live expression", which stays in place even after refreshes. You can then watch the summary change as resources are fetched and received.
